website/docs/r/service.html.markdown

```diff
@@ -454,9 +444,6 @@ the extra mount mappings for the container. Each `configs` is a reference to a s
 * `limits` - (Optional, list of strings) Describes the resources which can be advertised by a node and requested by a task.
   * `nano_cpus` (Optional, int) CPU shares in units of 1/1e9 (or 10^-9) of the CPU. Should be at least 1000000
   * `memory_bytes` (Optional, int) The amount of memory in bytes the container allocates
-  * `generic_resources` (Optional, map) User-defined resources can be either Integer resources (e.g, SSD=3) or String resources (e.g, GPU=UUID1)
-    * `named_resources_spec` (Optional, set of string) The String resources, delimited by `=`
-    * `discrete_resources_spec` (Optional, set of string) The Integer resources, delimited by `=`
```

Comment:
This PR includes breaking changes, right?

Reply:
Yes, it will include BCs and I guess #102 as well. So we will bump a major.

Reply:
I see.

Reply:
📝 I don't know the feature `generic_resources` well. I'll investigate later. 

* https://docs.docker.com/engine/reference/commandline/service_create/#create-services-requesting-generic-resources
* https://pkg.go.dev/github.com/docker/docker@v20.10.5+incompatible/api/types/swarm#GenericResource
* https://pkg.go.dev/github.com/docker/docker@v20.10.5+incompatible/api/types/swarm#ResourceRequirements